dashboard/app/models/concerns/concept_difficulties.rb

```diff
@@ -0,0 +1,23 @@
+module ConceptDifficulties
+  extend ActiveSupport::Concern
```

Comment:
Could this just be a non-`Concern` `module`, or are we using the specific `Concern` functionality here (mostly involving mixing in instance-methods into other classes, if I recall)?